providers/shodan/resources/shodan.lr.manifest.yaml

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) Mondoo, Inc.
+# SPDX-License-Identifier: BUSL-1.1
+
+resources:
+  shodan:
+    fields:
+      field: {}
+    min_mondoo_version: 9.0.0
+  shodan.apiPlan:
+    fields:
+      monitoredIps: {}
+      plan: {}
+      scanCredits: {}
+      telnet: {}
+      unlocked: {}
+      unlockedLeft: {}
+    min_mondoo_version: 9.0.0
+  shodan.domain:
+    fields:
+      name: {}
+      nsrecords: {}
+      subdomains: {}
+      tags: {}
+    min_mondoo_version: 9.0.0
+  shodan.host:
+    fields:
+      asn: {}
+      hostnames: {}
+      ip: {}
+      isp: {}
+      org: {}
+      os: {}
+      ports:
+        min_mondoo_version: latest
```

Comment:
nit: these can get collapsed down to just {} right?

Reply:
+1